src/sympc/tensor/mpc_tensor.py

```diff
@@ -33,6 +33,7 @@
     "squeeze",
     "unsqueeze",
     "view",
+    "expand",
```

Comment:
### Part 2.2
... to be added here as well right?

Reply:
Hmm, they were already added - a few lines below, but we don't need them in the api.py, because those are used only if you want to call something remotely, but we call them locally